ContentProtection/OfflinePlayReadyAndWidevine/OfflinePlayReadyAndWidevine.csproj

```diff
@@ -1,37 +1,36 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
-    <StartupObject />
-  </PropertyGroup>
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net7.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+	</PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <LangVersion>latest</LangVersion>
-  </PropertyGroup>
+	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
This property group will likely break release builds.

Reply:
I don't understand why it would break release builds. I just tested. Changes are just a formatting change and this two new lines
<ImplicitUsings>enable</ImplicitUsings>
<Nullable>enable</Nullable>

Reply:
Removed